compiler/cli.py

```diff
@@ -53,7 +53,7 @@ def make_command_line_parser():
         help='Root directory of all relative paths in manifest file.',
         default=os.getcwd())
     parser.add_argument(
-        '--outputpar',
+        '--output_par',
```

Comment:
wouldn't this be a backwards-incompatible change?

Reply:
I was under the impression that the compiler was only ever called from the subpar.bzl file (besides the tests) and is not intended for manual invokation as it’s so tightly wedded to the output of a py_binary rule, making this more of an internal implementation detail.

I can back out this change if desired though. If we were targeting more recent versions of python we could have the both versions with alias, but that's not available in 2.7

Reply:
I don't think the compiler is a public API. It's certainly not documented as such.